packages/components/pie-link/test/visual/pie-link.spec.ts

```diff
@@ -0,0 +1,14 @@
+
+import { test } from '@sand4rt/experimental-ct-web';
+import percySnapshot from '@percy/playwright';
+import { PieLink, LinkProps } from '@/index';
+
+test.describe('PieLink - Visual tests`', () => {
```

Comment:
In future I wonder if we should comment the percy tests out on newly generated components. They don't really give us anything and would save on usage

Reply:
We could add a X which would stop this running.

Reply:
yea that makes sense and happy to it part of this PR
@siggerzz @JoshuaNg2332 is that okay?

Reply:
Makes sense to me @jamieomaguire